src/Network/Minio/S3API.hs

```diff
@@ -435,3 +439,30 @@ getBucketNotification bucket = do
 -- | Remove all notifications configured on a bucket.
 removeAllBucketNotification :: Bucket -> Minio ()
 removeAllBucketNotification = flip putBucketNotification def
+
+-- | Fetch the policy if any on a bucket.
+getBucketPolicy' :: Bucket -> Minio Text
+getBucketPolicy' bucket = do
```

Comment:
We can just rename this function to `getBucketPolicy` and make sure it is exported in this module.